service/Makefile.am

```diff
@@ -122,6 +127,12 @@ geopmwrite_SOURCES = src/geopmwrite_main.cpp
 geopmwrite_CXXFLAGS = $(AM_CXXFLAGS) -std=c++17
 geopmwrite_LDADD = libgeopmd.la
 
+if ENABLE_FUZZTESTS
```

Comment:
Maybe better to put this change in ``service/integration/Makefile.mk`` and move ``service/src/fuzztest_geopmhash_main.cpp`` to ``service/integration/fuzztest_geopmhash_main.cpp``?

Reply:
Will do